source/database-connections/database-connections.txt

```diff
@@ -19,21 +19,42 @@ You can save this information as a database connection.
 Use Cases
 ---------
 
-You can save connection settings for relational and MongoDB databases
-when creating projects and migration jobs. You can use these saved database
-connections for any task that requires connecting to a relational or MongoDB
-database.
+You can save a new database connection:
+
+- When creating a project from a live database connection
+
+- When creating or modifying a data migration job
+
+- From the :guilabel:`Database Connections Bar` in a project
+
+- From the Relational Migrator :guilabel:`Connections` page
+
+After saving a database connection, you can use it for any 
+task that requires connecting to a database.
+
+Supported Alternatives
+----------------------
+
+Relational Migrator can use Data Definition Language (DDL) files 
+exported from your relational database to create a project when 
+network connectivity or permissions to your relational database 
+are a consideration.
```

Comment:
[Suggestion]

Start with the condition, so users can tell whether they should care about this information. As in something like:

"If network connectivity or relational database permissions are a consideration, you can use Data Definition Language (DDL) files exported from your relational database to create a Relational Migrator project."